### ISSUE TITLE
One of tested-with?

### ISSUE THREAD

**Issue by philderbeast** [ucsd-progsys/liquid-fixpoint#564]
Are we testing with all `tested-with` versions of GHC?

https://github.com/ucsd-progsys/liquid-fixpoint/blob/db2c3c7ce116cf953d361e04e918cd3be0732c31/liquid-fixpoint.cabal#L24

I've looked at `.circleci/config.yml` and it seems to be doing one `stack build` so picking up `ghc-8.10.7` from `resolver: lts-18.27`.

I'm testing with Z3 and multiple GHC versions as a github action for [plugins-for-blobs](https://github.com/BlockScope/plugins-for-blobs) if that helps.

**Comment by philderbeast** [comment]
Could we go with the latest in each series (dropping < 8):

```diff
- tested-with: GHC == 7.10.3, GHC == 8.0.1, GHC == 8.4.3, GHC == 8.6.4, GHC == 8.10.1, GHC == 8.10.7
+ tested-with: GHC == 8.0.2, GHC == 8.4.4, GHC == 8.6.5, GHC == 8.8.4, GHC == 8.10.7, GHC == 9.0.2, GHC == 9.2.2
```

**Comment by ranjitjhala** [comment]
I think it’s fine to drop everything < 8.8

**Comment by facundominguez** [comment]
Closing this since we merged #591. Feel free to reopen if there is more to do.
